config/app.config

```diff
@@ -64,6 +64,7 @@ ehcache.multicast.port = "7077"
 ehcache.peerDiscovery = "PubSub"
 redis.host = "redis-srv"
 ehcache.replicate = true
+lsst.hips.masterUrl="https://irsa.ipac.caltech.edu/data/hips/list"
```

Comment:
This line needs a comment saying that this is pointing to IRSA to permit testing outside the auth environment, and needs to be overridden in the RSP deployment configurations.